src/main/java/io/netty/incubator/codec/quic/QuicheQuicChannel.java

```diff
@@ -813,6 +813,11 @@ private boolean handleWritableStreams() {
                             if (streamChannel != null) {
                                 int capacity = Quiche.quiche_conn_stream_capacity(connAddr, streamId);
                                 if (capacity < 0) {
+                                    if (capacity == Quiche.QUICHE_ERR_STREAM_STOPPED) {
+                                        // Streams that received STOP_SENDING and no longer writable
+                                        // but should be not closed.
+                                        break;
```

Comment:
shouldn't this say `continue` as we still want to process all the other streams ?

```suggestion
                                        continue;
```